### PR TITLE
tests: kubecost, kubevirt, linkerd, nginx tests

### PR DESCRIPTION
**Pull request by mintbomb27**
## Description
Contains 13 kuttl tests for KubeVirt(1), KubeCost(2), Linkerd(7), Nginx-Ingress(3) Policies.

## Checklist

- [x] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [x] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [x] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

### CODE REVIEW

**Comment by chipzoller** [comment]
I see a failure in /istio/create-authorizationpolicy. You may need to add a `sleep 3` statement after creating the ClusterRole to give Kyverno a bit more time to pick up the permissions.

**Comment by chipzoller** [comment]
Let me know when ready for final review.

**Comment by chipzoller** [comment]
Ok, try specifying the Istio one in GVK format.

**Review by chipzoller — CHANGES_REQUESTED** [review]
Nice work! One very minor permissions update and that's all.

File: temp.json
[the remark this answers is not in the document]
Comment (mintbomb27) [reply temp.json:1]: Really sorry. That was to force delete a namespace. Removing it.

File: kubevirt/add-services/01-clusterrole.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+kind: ClusterRole
+metadata:
+  name: kyverno:background-controller:kubevirt-services
+  labels:
+    app.kubernetes.io/component: background-controller
+    app.kubernetes.io/instance: kyverno
+    app.kubernetes.io/part-of: kyverno
+rules:
+- apiGroups:
+  - "*"
```
[the remark this answers is not in the document]
Comment (mintbomb27) [reply kubevirt/add-services/01-clusterrole.yaml:11]: Got it! Changing.

**Review by chipzoller — APPROVED** [review]

**Pull request merged**